metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/ProfileConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.common.server.util;
+
+/**
+ * Profile Constants.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+public final class ProfileConstants {
```

Comment:
I think we talked about this in person but I don't like the idea of baking "profiles" which is a configuration construct into the code like this. It says something is wrong in the architecture of the code. We should fix the problem causing the need for this class and get rid of it.